### PR TITLE
feat: support configuring eslint command flags

### PR DESCRIPTION
**Pull request by timothymathison**
Adds an option to disable `--fix` flag and an option to provide additional flags for the `eslint` command run by the `eslint` task.

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by timothymathison** [comment]
@mrgrain any idea why there's test failures for windows - unrelated to my changes?

**Review by mrgrain — CHANGES_REQUESTED** [review]
We need tests for this please.

File: src/javascript/eslint.ts [hunk 1]
```diff
@@ -96,6 +101,19 @@ export interface EslintOptions {
   readonly yaml?: boolean;
 }
 
+export interface EslintCommandOptions {
```
Comment (mrgrain) [inline src/javascript/eslint.ts:104]: Let's flatten this and include it in `EslintOptions`

File: src/javascript/eslint.ts [hunk 2]
```diff
@@ -96,6 +101,19 @@ export interface EslintOptions {
...
+  /**
+   * Whether to include --fix flag in base eslint task
```
Comment (mrgrain) [inline src/javascript/eslint.ts:106]:
```suggestion
   * Whether to fix eslint issues when running the eslint task
```

File: src/javascript/eslint.ts [hunk 3]
```diff
@@ -96,6 +101,19 @@ export interface EslintOptions {
...
+   * @default true
+   */
+  readonly fix?: boolean;
+
+  /**
+   * Extra flag arguments to pass to eslint command
+   */
+  readonly extraFlagArgs?: string[];
```
Comment (mrgrain) [inline src/javascript/eslint.ts:114]:
```suggestion
  /**
   * Additional arguments to pass to eslint command line.
   */
  readonly extraArgs?: string[];
```

**Review by mrgrain — CHANGES_REQUESTED** [review]
Thanks for the tests! Just some tiny changes to the API and descriptions.

**Review by mrgrain — APPROVED** [review]

**Pull request merged**